src/System.Management.Automation/resources/ExtendedTypeSystem.resx

```diff
@@ -370,4 +370,16 @@ PS&gt; [System.Collections.Generic.Comparer``1]::get_Default()
   <data name="InvalidCastExceptionNonCoreType" xml:space="preserve">
     <value>Cannot convert value to type "{0}". Only core types are supported in this language mode.</value>
   </data>
+  <data name="InvalidCastToByRefLikeType" xml:space="preserve">
+    <value>Cannot convert to the ByRef-like type "{0}". ByRef-like types cannot be boxed and should be used only on stack.</value>
```

Comment:
`should` implies other uses are possible. You could keep it simple and just say those types are not supported in PowerShell because it might not be obvious to most that PowerShell boxes everything.

Reply:
Will update the message to `... ByRef-like types are not supported in PowerShell`.